recipes/deps/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "deps" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/ESSS/{{ name }}.git
+  git_rev: v{{ version }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  entry_points:
+    - deps = deps.deps_cli:main_func
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - pip
+  run:
+    - python
+    - click
+    - colorama
+    - jinja2
+    - pyyaml
+    - futures  # [py27]
+
+test:
+  requires:
+    - pytest >=3
+    - pytest-mock
+    - future
+  imports:
+    - deps
+  commands:
+    - deps --help
+    - deps --version
+    - python -m deps --version
+    - pytest source/python
+  source_files:
+    - source/python
+
+about:
+  home: https://github.com/ESSS/deps
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'An utility to execute commands on multiple projects.'
+  doc_url: https://github.com/ESSS/deps
+  dev_url: https://github.com/ESSS/deps
+
+extra:
+  recipe-maintainers:
+    - "tadeu"
+    - "nicoddemus"
+    - "prusse-martin"
```

Comment:
I don't think you meed the quotes here.

Reply:
True, I was trying to see if this made the "Recipe maintainers should be a json list." bug go away :)